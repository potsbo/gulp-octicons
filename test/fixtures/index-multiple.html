<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <title>Index</title>
  </head>
  <body>
    <!-- octicons:arrow-down -->
    <!-- endocticons -->

    <!-- octicons:arrow-left -->
    <!-- endocticons -->

    <!-- octicons:arrow-right -->
    <!-- endocticons -->

    <!-- octicons:arrow-up -->
    <!-- endocticons -->
  </body>
</html>
