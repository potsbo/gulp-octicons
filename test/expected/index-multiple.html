<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <title>Index</title>
  </head>
  <body>
    <!-- octicons:arrow-down -->
    <svg version="1.1" width="10" height="16" viewBox="0 0 10 16" class="octicon octicon-arrow-down" aria-hidden="true"><path fill-rule="evenodd" d="M7 7V3H3v4H0l5 6 5-6H7z"/></svg>
    <!-- endocticons -->

    <!-- octicons:arrow-left -->
    <svg version="1.1" width="10" height="16" viewBox="0 0 10 16" class="octicon octicon-arrow-left" aria-hidden="true"><path fill-rule="evenodd" d="M6 3L0 8l6 5v-3h4V6H6V3z"/></svg>
    <!-- endocticons -->

    <!-- octicons:arrow-right -->
    <svg version="1.1" width="10" height="16" viewBox="0 0 10 16" class="octicon octicon-arrow-right" aria-hidden="true"><path fill-rule="evenodd" d="M10 8L4 3v3H0v4h4v3l6-5z"/></svg>
    <!-- endocticons -->

    <!-- octicons:arrow-up -->
    <svg version="1.1" width="10" height="16" viewBox="0 0 10 16" class="octicon octicon-arrow-up" aria-hidden="true"><path fill-rule="evenodd" d="M5 3L0 9h3v4h4V9h3L5 3z"/></svg>
    <!-- endocticons -->
  </body>
</html>
